ktor-utils/common/test/io/ktor/util/ConcurrentListTest.kt

```diff
@@ -37,4 +37,9 @@ class ConcurrentListTest {
             assertEquals(it, list[it])
         }
     }
+
+    @Test
+    fun testClear() {
+        ConcurrentList<Unit>().clear()
```

Comment:
This test doesn't have any assertions. Do you expect that some exception won't be thrown?

Reply:
Yep. There was exception in this method

Reply:
Can we illustrate that no exception is expected?